src/einsteinpy/shadow/core.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np
+import scipy as si
+import matplotlib.pyplot as plt
+from scipy import optimize
+from scipy import integrate
+
+# ds^2 = -(1-2*M/r)dt^2 + 1/(1-2*M/r)dr^2 + r^2*(d\theta^2 + sin(\theta)^2*d\phi^2)
+M = 1
+
+# Number of rays (n_rays) with the impact Paramter B
+n_rays = 5000  # No. of rays
+bcrit = 3 * np.sqrt(3) * M  # Critical Impact Paramter
+bcrit = 5.196152422706632
```

Comment:
Why is this redifined?

Reply:
No need for redefining that. It was just meant to show the value of critical impact parameter for M = 1.